dio/test/interceptor_test.dart

```diff
@@ -349,6 +349,40 @@ void main() {
         expect(response.requestOptions.contentType, isNull);
       });
 
+      test("Interceptor gets stacktrace in onError", () async {
+        final dio = Dio();
+        dio.options.baseUrl = EchoAdapter.mockBase;
+        dio.httpClientAdapter = EchoAdapter();
+
+        late final StackTrace caughtStackTrace;
```

Comment:
Use a nullable variable to avoid tests failing with more complexity.

Reply:
My idea was that it should fail if `caughtStackTrace` wasn't initialized.
Should I handle it by checking against null in the expect instead?

Reply:
The idea is to avoid `LateInitializationError` causing unreadable logs during tests.